src/browser/index.jsx

```diff
@@ -219,30 +219,16 @@ var TabBar = React.createClass({
     var tabs = this.props.teams.map(function(team, index) {
       var unreadCount = 0;
       var badgeStyle = {
-        unread: {
-          background: '#383838',
-          float: 'right',
-          color: 'white',
-          textAlign: 'center',
-          marginTop: '5px',
-          width: '10px',
-          height: '10px',
-          marginLeft: '5px',
-          borderRadius: '50%',
-        },
-
-        mention: {
-          background: '#f1342c',
-          float: 'right',
-          color: 'white',
-          minWidth: '19px',
-          fontSize: '12px',
-          textAlign: 'center',
-          lineHeight: '20px',
-          height: '19px',
-          marginLeft: '5px',
-          borderRadius: '50%',
-        }
+        background: '#FF1744',
```

Comment:
It might be a little thing, the `background` was changed to Material Red A400 (#FF1744).
@asaadmahmood @jasonblais In other part (Red badge in taskbar and tray), Red A400 is used now. Would you hear opinion?